crawl-ref/source/tilesdl.cc

```diff
@@ -897,6 +897,7 @@ void TilesFramework::do_layout()
     m_region_tile->dx = m_viewsc.x;
     m_region_tile->dy = m_viewsc.y;
     int message_y_divider = 0;
+    int sidebar_pw;
```

Comment:
I'm traveling for a few more days and so won't have time to actually look at this soon, but while restarting an errored travis job I noticed that sidebar_pw remains uninitialized if use_small_layout=true.  (From a compiler warning that might not show up for all compilers.)

Reply:
Good catch